platforms/ios/src/TangramMap/TGHelpers.mm

```diff
@@ -36,10 +36,14 @@ + (TGError)convertTGErrorTypeFrom:(Tangram::Error)error
             return TGErrorSceneUpdatePathNotFound;
         case Tangram::Error::scene_update_value_yaml_syntax_error:
             return TGErrorSceneUpdateValueYAMLSyntaxError;
+        case Tangram::Error::no_valid_scene:
+            return TGErrorNoValidScene;
+        case Tangram::Error::none:
+            return TGErrorNone;
     }
 }
 
-+ (NSError *)errorFromSceneUpdateError:(Tangram::SceneUpdateError)updateError
++ (NSError *)errorFromSceneError:(Tangram::SceneError)updateError
```

Comment:
Yeah that's fine, we don't have to fix it in this PR but I wanted to note the issue.